addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -577,6 +577,8 @@
                     $target: $content,
                 });
             }
+            let websiteAnimateWidget = new publicWidget.registry.WebsiteAnimate();
+            websiteAnimateWidget.attachTo($('#wrapwrap'));
```

Comment:
Multiple questions on this one:
- Why do we always create it if it's only an issue for the "webpage" slides?
- Isn't it a bit much to attach it to "wrapwrap" instead of locally to the "wpContainer"?
- If we open multiple web pages in a row, this will create successive instances of "WebsiteAnimate", isn't that an issue? Shouldn't we store the current instance in a "this" variable and properly destroy it when the next slide is loaded?

Reply:
Hello @awa-odoo , thank you for the review, I've made some changes, let me know if they make sense.